projects/packages/my-jetpack/src/products/class-scan.php

```diff
@@ -114,47 +112,6 @@ public static function get_wpcom_product_slug() {
 		return 'jetpack_scan';
 	}
 
-	/**
-	 * Hits the wpcom api to check scan status.
-	 *
-	 * @todo Maybe add caching.
-	 *
-	 * @return Object|WP_Error
-	 */
-	private static function get_state_from_wpcom() {
```

Comment:
We weren't using the scan class really at all, this probably wasn't being called in any scenario, but just in case, I figured I'd remove it